src/dmd/backend/backconfig.d

```diff
@@ -74,6 +76,29 @@ version (MARS)
     tytab[TYchar] |= TYFLuns;
     bool mscoff = model & 1;
     model &= 32 | 64;
+
+    if (dwarf)
+    {
+        bool supported;
+        foreach (v; DWARF_SUPPORTED_VERSIONS)
+        {
+            if (v == dwarf)
+                supported = true;
+        }
+
+        if (!supported)
+        {
+            import dmd.backend.errors;
+            error(null, 0, 0, "DWARF version %d is currently supported by DMD !", dwarf);
```

Comment:
Please, no `!` in error messages. No need for `currently` or `dmd` either, these are all implicit. Missing `not`.

Reply:
Use `%u` for unsigned integral types.